include/fluent-bit/flb_parser.h

```diff
@@ -50,6 +50,7 @@ struct flb_parser {
     int time_offset;      /* fixed UTC offset */
     int time_keep;        /* keep time field */
     int time_strict;      /* parse time field strictly */
+    int no_bare_keys;     /* in logfmt parsers, require all keys to have values */
```

Comment:
Could you rename like `logfmt_no_bare_keys` ?
This parameter is specific for logfmt.